app/services/api/teachers/withdraw.rb

```diff
@@ -24,6 +24,8 @@ def withdraw
 
         record_withdraw_event!
       end
+
+      teacher
```

Comment:
Do we need a `reload` here for the serializer to return the withdrawal details in the response? If we do, can you think about how we could cover that more generally in the request spec - it might just need this:

```
expect(response.body).to eq(serializer.render(resource_type.last, root: "data", **options))
```

To be changed to:

```
expect(response.body).to eq(serializer.render(resource_type.last.reload, root: "data", **options))
```

That way we're expecting it to equal the latest version of the updated object, in a serialized format

Reply:
cool, thanks for adding that; I think that will do it, but can you make sure to eyeball the withdrawn response from postman as well to double check